### DISCUSSION TITLE
Yesterday my code is working with json file but today give the error below

### DISCUSSION THREAD

**Discussion by ahmedbilal360** [freekmurze/laravel-analytics#491] [General]
{ "reason": "ACCESS_TOKEN_EXPIRED", "domain": "googleapis.com", "errorInfoMetadata": { "service": "analyticsdata.googleapis.com", "method": "google.analytics.data.v1beta.BetaAnalyticsData.RunReport" }, "message": "Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https:\/\/developers.google.com\/identity\/sign-in\/web\/devconsole-project.", "code": 16, "status": "UNAUTHENTICATED", "details": [ { "@type": "type.googleapis.com\/google.rpc.ErrorInfo", "reason": "ACCESS_TOKEN_EXPIRED", "domain": "googleapis.com", "metadata": { "service": "analyticsdata.googleapis.com", "method": "google.analytics.data.v1beta.BetaAnalyticsData.RunReport" } } ] }

Please help me to resolve this issue

**Comment by danikaharu** [comment]
I also have the same problem. I have been using version 5. Can anyone provide a solution?

**Comment by mahmoudegh** [comment]
i have the same issue too .. any help ?
